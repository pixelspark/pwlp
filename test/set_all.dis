0000.	e0	USER	get_length
0001.	20	PEEKB	0
0002.	11	PUSHB	[01]
0004.	81	BINARY	SUB
0005.	11	PUSHB	[ff]
0007.	11	PUSHB	[ff]
0009.	85	BINARY	AND
0010.	10	PUSHB	0
0011.	11	PUSHB	[ff]
0013.	85	BINARY	AND
0014.	74	UNARY	SHL8
0015.	86	BINARY	OR
0016.	10	PUSHB	0
0017.	11	PUSHB	[ff]
0019.	85	BINARY	AND
0020.	74	UNARY	SHL8
0021.	74	UNARY	SHL8
0022.	86	BINARY	OR
0023.	e3	USER	set_pixel
0024.	01	POP	1
0025.	71	UNARY	DEC
0026.	60	JNZ	to 1
0029.	01	POP	1
0030.	e4	USER	blit

