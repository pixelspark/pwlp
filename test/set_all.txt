for(n=get_length){
	set_pixel(n-1,255,0,0)
};
blit;