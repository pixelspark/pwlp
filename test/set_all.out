ff0000 ff0000 ff0000 ff0000 ff0000 ff0000 ff0000 ff0000 ff0000 ff0000 
